packages/tiled/lib/src/layer.dart

```diff
@@ -464,6 +502,32 @@ class ObjectGroup extends Layer {
   }) : super(
           type: LayerType.objectGroup,
         );
+
+  @override
+  ExportElement export() => ExportElement(
+        'objectgroup',
+        {
+          'id': id?.toExport(),
```

Comment:
You're ok with this being "id": null? Same for `class` below?

Reply:
No not really, but as it stands it is nullable in the class. So if it is allowed to import with a null value it think it should be possible to export with a null value

Reply:
Yeah, so I was wondering if I don't have an ID in my tiled file and then I export, I get a null id; will that break someone else.  If its not present, we could just add it to the map as a map-if:
{
    if (id != null) 'id':  id!.toExport(),  
}

same for other nullables.